deploy/rbac-clowdapp.yml

```diff
@@ -985,6 +985,22 @@ objects:
           volumeMounts:
           - mountPath: /tmp
             name: users-data
+      - name: force-bootstrap-tenants
+        podSpec:
+          image: quay.io/cloudservices/rbac:latest
+          command:
+            - /opt/rbac/rbac/manage.py
+            - bootstrap_tenants
+            - "--all"
+            - "--force"
+          resources:
+            limits:
+              cpu: 300m
+              memory: 1Gi
+            requests:
+              cpu: 50m
+              memory: 512Mi
+          env: *job_common_env_vars
```

Comment:
These envs are outdated. In my PR, I added the extra required envs to workspace_job_env_vars, we can use that here